chia/rpc/wallet_rpc_api.py

```diff
@@ -1699,16 +1701,41 @@ async def get_offer_summary(self, request: Dict[str, Any]) -> EndpointResult:
         offered, requested, infos = offer.summary()
 
         if request.get("advanced", False):
-            return {
+            response = {
                 "summary": {"offered": offered, "requested": requested, "fees": offer.fees(), "infos": infos},
                 "id": offer.name(),
             }
         else:
-            return {
+            response = {
                 "summary": await self.service.wallet_state_manager.trade_manager.get_offer_summary(offer),
                 "id": offer.name(),
             }
 
+        # This is a bit of a hack in favor of returning some more manageable information about CR-CATs
+        # A more general solution surely exists, but I'm sure what it is right now
+        return {
```

Comment:
If it is not a CRCAT offer, will the response be exactly the same as the previous response? I'm worried that this change will cause issues on DEX or NFT exchange.

Reply:
Yeah the hack only triggers if there's exactly a key called "proofs_checker" that is under exactly a key called "also" in the driver dict of the offer being summarized.  In addition, we're only adding a key, which should break any existing parsing anyways.

Reply:
If we can remove the "infos" when it is not a CRCAT it will be better.

Reply:
Infos is something that's already in the request.  It gives information about the assets involved in the trade.  I'm just adding one extra key to the info for a CR-CAT.